Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.h

```diff
@@ -195,6 +195,7 @@ namespace AzToolsFramework
         // but can be overridden here
         void SetSpawnDebugView(bool shouldSpawn);
 
+        static AZStd::string GetEnableDPECVarName() { return "ed_enableDPE"; }
```

Comment:
This should return a `const char*`.
No need to create an `AZStd::string` before it is needed

Reply:
yep, done and done.